airbyte-integrations/connectors/destination-mssql/src/test-integration/kotlin/io/airbyte/integrations/destination/mssql/v2/MSSQLWriterTest.kt

```diff
@@ -295,6 +303,14 @@ internal class BulkInsert :
             ) { spec -> MSSQLConfigurationFactory().makeWithOverrides(spec, emptyMap()) },
     ) {
 
+    @Disabled(
+        "temporarily disabling while I work on implementing better type handling in bulk inserts - https://github.com/airbytehq/airbyte-internal-issues/issues/12128 / https://github.com/airbytehq/airbyte/pull/55884"
+    )
+    @Test
+    override fun testUnknownTypes() {
+        super.testUnknownTypes()
+    }
```

Comment:
reenabling test in https://github.com/airbytehq/airbyte/pull/55884/commits/e18d1d0eddf8bb8c91d220e25f2a9823dc598a72